towerofhanoi.c

```diff
@@ -1,17 +0,0 @@
-#include<stdio.h>
-void TowerOfHanoi(int size, char c1, char c2, char c3)
-{
-    if(SIZE==1)
-    {

```

Comment:
Why did you delete file? simply could have commited it

Reply:
will keep that in mind next time